PennMobile/Home/Cells/Calendar/CalendarAPI.swift

```diff
@@ -30,19 +30,21 @@ class CalendarAPI: Requestable {
                 var events = [CalendarEvent]()
                 for json in jsonArray {
                     let name = json["name"].stringValue
-                    let start = json["start"].stringValue
-                    let end = json["end"].stringValue
+
+                    //set time to 6 am EST
+                    let start = "\(json["start"].stringValue) 6:00 AM"
+                    let end = "\(json["end"].stringValue) 6:00 AM"
                     
                     let dateFormatter = DateFormatter()
-                    dateFormatter.dateFormat = "yyyy-MM-dd"
+                    dateFormatter.dateFormat = "yyyy-MM-dd h:mm a"
                     dateFormatter.timeZone = TimeZone(abbreviation: "EST")
+
```

Comment:
I'm not exactly getting why the addition/removal of the hour portion of the time string is causing/fixing the bug.

Reply:
@01jongmin Yeah, I think the problem is that when it parses the date from https://api.pennlabs.org/calendar/, it sets the time to 12:00 AM (since the API does not specify the time). And 12:00 am on Aug 24 in EST = 9:00 pm on Aug 23 in PST (and in other timezones too). So, if I manually set the time to 6:00 am instead of 12:00 am, then it will show aug 24 in more timezones.

Still feels like a bit of a hack, but seems to work for almost all timezones. If you have other ideas, shoot.

Reply:
Oh got it. That makes sense. It does feel a bit like a hack, but I don't think it will lead to any breaking problems down in the future for us. LGTM!